src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReplacementServiceTests.cs

```diff
@@ -3145,6 +3164,15 @@ await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbC
                 Assert.NotNull(chartLegendItem.DataSet.Location);
                 Assert.Equal(replacementLocation.Id, chartLegendItem.DataSet.Location!.Value);
 
+                var chartDataSetConfigs = replacedDataBlock.Charts[0].Map;
+                Assert.NotNull(chartDataSetConfigs);
+                var chartDataSetConfig = Assert.Single(chartDataSetConfigs);
+                var filterId = Assert.Single(chartDataSetConfig.DataSet.Filters);
+                Assert.Equal(replacementFilterItem1.Id, filterId);
+                Assert.Equal(replacementIndicator.Id, chartDataSetConfig.DataSet.Indicator);
+                Assert.NotNull(chartLegendItem.DataSet.Location);
+                Assert.Equal(replacementLocation.Id, chartLegendItem.DataSet.Location!.Value);
```

Comment:
Why are these assertions here? We're looking at data set configs so these are irrelevant and have already been assert earlier anyway.

It seems like we should actually be asserting on the data set configs instead?

Reply:
I copied and pasted from above and missed that I'd not changed these to test on data set config rather than the legend - I do that often enough that I wonder whether copying and pasting isn't worth the risk...

Reply:
Tidied it up in d074f5d34bcf7059385006bb17f3bd2f7448d80c